src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
+  @ApiProperty({
+    example: 'username',
+  })
+  @IsNotEmpty({ message: 'First name is required' })
+  name: string;
```

Comment:
PullProofAI

The property `name` is annotated as a first name, but the field name suggests it might be a username. Consider updating the annotation or field name for clarity.
```diff
-   @IsNotEmpty({ message: 'First name is required' })
+   @IsNotEmpty({ message: 'Username is required' })
```


<!-- This is an auto-generated comment by PullProofAI -->